src/components/app.jsx

```diff
@@ -84,9 +90,22 @@ class App extends React.Component {
     }
 
     render() {
+        const cardsList = this.state.forecasts.map((singleForecast) => <WeatherCard 
```

Comment:
Create weather results component that will handle creating weather cards and their layout